test/service/test_manifest.py

```diff
@@ -1616,48 +1616,49 @@ def test_hash_validity(self):
+                                         args=dict(catalog=self.catalog,
+                                                   format=format_.value,
+                                                   filters='{}',
+                                                   objectKey=object_key)).url
+                manifest_url = 'https://url.to.manifest?foo=bar'
+                manifest = Manifest(location=manifest_url,
                                     was_cached=False,
                                     format_=format_,
                                     catalog=self.catalog,
                                     filters={},
                                     object_key=object_key)
                 get_cached_manifest.return_value = None, manifest
-                # Request the fetch manifest endpoint to verify the response
                 request_url = furl(self.base_url,
-                                   path='/fetch/manifest/files',
-                                   args=dict(format=format_.value,
-                                             filters='{}'))
-                response = requests.get(request_url.url)
-                response_json = response.json()
-                expected_json = {
-                    'Status': 302,
-                    'Location': url,
-                }
-                if format_ == ManifestFormat.curl:
-                    expected_json['CommandLine'] = {
-                        'cmd.exe': f'curl.exe --location "{url}" | curl.exe --config -',
-                        'bash': f"curl --location '{url}' | curl --config -"
-                    }
-                self.assertEqual(expected_json, response_json, response.content)
+                                   path='/manifest/files',
+                                   args=dict(format=format_.value, filters='{}'))
+                with self.subTest(format_=format_, fetch=fetch):
+                    if fetch:
+                        request_url.path.segments.insert(0, 'fetch')
+                        response = requests.get(request_url.url).json()
+                        expected = {
+                            'Status': 302,
+                            'Location': azul_manifest_url,
+                        }
+                        if format_ == ManifestFormat.curl:
+                            expected['CommandLine'] = {
+                                'cmd.exe': f'curl.exe --location "{azul_manifest_url}" | curl.exe --config -',
+                                'bash': f"curl --location '{azul_manifest_url}' | curl --config -"
+                            }
+                        self.assertEqual(expected, response)
+                    else:
+                        response = requests.get(request_url.url, allow_redirects=False)
+                        self.assertEqual(302, response.status_code)
+                        self.assertEqual(manifest_url, response.headers['location'])
```

Comment:
I was wondering the same thing while working on this ticket, I could only reason that `/fetch` endpoints are useful only to the Data Browser. And because of it, only DB users might require a command for programatic access, but I am not sure how true that is. Maybe a question for @hannes-ucsc?

Reply:
I'll leave this unresolved. I'll leave it up to you if you want to decide to bring this up with him or not.